### PR TITLE
Updated Browsing and Downloading Page

### PR DESCRIPTION
**Pull request by chack1172**
Updated Browsing and Downloading page to the new layout of extend website.

### CODE REVIEW

**Comment by Ben-MyBB** [comment]
Looks good to me. Thanks for the contribution.

**Review by Ben-MyBB — APPROVED** [review]

**Comment by Shade-** [comment]
There are some language mistakes here and there though. Would be nice if a native speaker addressed them.

**Comment by Ben-MyBB** [comment]
@Shade- Looks fine to me from a quick scan? @andrewjs18

File: extend/browsing-downloading-from-extend.md [hunk 1]
```diff
@@ -10,66 +10,78 @@ categories: [extend]
...
 
 
 ### Browsing a category
 
-[![Browsing and Filtering](/assets/images/mods/downloaders1.png)](/assets/images/mods/downloaders1.png)
+When you visit a category page, here's how it looks like:
+
+[![Browsing and Filtering](/assets/images/extend/downloaders1.png)](/assets/images/extend/downloaders1.png)
+
+For each projects you can view the cover and this informations:
```
Comment (Shade-) [inline extend/browsing-downloading-from-extend.md]: For each **project** you can view the cover and **these** informations

File: extend/browsing-downloading-from-extend.md [hunk 2]
```diff
@@ -10,66 +10,78 @@ categories: [extend]
...
 - Bug Tracker (you will see this option if it's been enabled by the project author)
 
 - Suggestions (you will see this option if it's been enabled by the project author)
 
-- Translations (translations posted by collaborators or the project owner)
-
+- License (where you can see the license of the project)
 
-If the owner of the project set a PayPal e-mail for donations, you will see a Donate button on the right as well.
+On the right side, you can see the latest build and a Donate button if the owner of the project set a PayPal e-mail for donations.
```
Comment (Shade-) [inline extend/browsing-downloading-from-extend.md]: ... if the owner of the project **sets**

**Review by Shade- — CHANGES_REQUESTED** [review]

**Comment by Shade-** [comment]
These are a couple of them. There may be others.

File: extend/browsing-downloading-from-extend.md (hunk 1 above)
Comment (Ben-MyBB) [reply extend/browsing-downloading-from-extend.md]: This doesn't make sense. I'd put- for each project you can view the cover and the following information:

File: extend/browsing-downloading-from-extend.md (hunk 2 above)
Comment (Ben-MyBB) [reply extend/browsing-downloading-from-extend.md]: Good spot.

File: extend/browsing-downloading-from-extend.md (hunk 1 above)
Comment (chack1172) [reply extend/browsing-downloading-from-extend.md]: You're right.

File: extend/browsing-downloading-from-extend.md (hunk 2 above)
Comment (chack1172) [reply extend/browsing-downloading-from-extend.md]: I think that set is correct (it's not present)

Comment (Ben-MyBB) [reply extend/browsing-downloading-from-extend.md]: I would say this sounds better:

On the right side, you can see the latest build and a Donate button if the owner of the project has set a PayPal e-mail for donations.

Comment (Shade-) [reply extend/browsing-downloading-from-extend.md]: Either you use present tense (sets) or present perfect (has set). I'd go with the latter.

Edit: Ninja'd

**Review by Shade- — APPROVED** [review]

**Comment by Ben-MyBB** [comment]
@Shade- This looks fine to me. Any further changes you'd like?

**Comment by Shade-** [comment]
There are still some issues. I will review this later when I come home.

File: extend/browsing-downloading-from-extend.md [hunk 3]
```diff
@@ -10,66 +10,78 @@ categories: [extend]
...
+- Stars
 
-Clicking the project names directs you towards the project page, on which you can see detailed information: full description, previews, license, suggestions, bugs, etc.
+- Downloads
 
+- Last Update
+
+There is also a download button that allows you to download the latest **stable** build.
+
+Clicking the project names or the Details button directs you towards the project page, on which you can see detailed information: full description, previews, license, suggestions, bugs, etc.
```
Comment (Shade-) [inline extend/browsing-downloading-from-extend.md]: Clicking the project **name**

File: extend/browsing-downloading-from-extend.md [hunk 4]
```diff
@@ -10,66 +10,78 @@ categories: [extend]
...
-- License (expands the license section)
+- Star (allows you to recommend projects)
 
-- Change Logs (page where the changes for each build are displayed)
+- Get Support (either the support link entered by the project author or a link to the support forum on the MyBB Community)
 
 - Report (where you can report a project)
 
-- Recommend (allows you to recommend projects)
+Under this buttons there is a tab with the following links:
```
Comment (Shade-) [inline extend/browsing-downloading-from-extend.md]: Under **these** buttons

**Review by Shade- — CHANGES_REQUESTED** [review]

File: extend/browsing-downloading-from-extend.md (hunk 3 above)
Comment (chack1172) [reply extend/browsing-downloading-from-extend.md]: Nice spot :)

File: extend/browsing-downloading-from-extend.md (hunk 4 above)
Comment (chack1172) [reply extend/browsing-downloading-from-extend.md]: You're right. My english is bad...

**Pull request merged**